docs/en/guides/flakes-guide.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Flakes
+---
+
+Flakes are one of the most complex and confusing concepts in Nix, especially for beginners. Even though, as dependency lock files, flakes aren't anything new, their use in the context of NixOS, an operating system, adds an unsurprising amount of complexity.
+
+However, understanding flakes is key for working with Nix systems effectively, and is a lot easier than often thought.
+
+## What are flakes?
+
+Flakes are an experimental feature of nix that adds two new important files, the `flake.nix` and `flake.lock`. While experimental, they are widely used among the Nix community. Flakes allow you to extend your Nix projects, including development shells, NixOS systems, home-manager installations, and more, with Nix code from other projects and repositories.
```

Comment:
Flakes do not "extend" anything. This is akin to claiming a PKGBUILD extends your project.

Clarify wording.